ballerina/sql_client.bal

```diff
@@ -96,13 +124,26 @@ public client class SQLClient {
         return resultStream;
     }
 
+    # Performs an SQL `SELECT` operation to read multiple records from the database when an advanced filter is provided.
+    #
+    # + filterClause - The filter query to be used (to be used in the SQL `WHERE` clauses)
+    # + rowType - The record-type to be retrieved (the entity record-type)
+    # + include - The relations to be retrieved (SQL `JOINs` to be performed)
+    # + return - A stream of records in the `rowType` type or an `error` if the operation fails
     public isolated function runExecuteQuery(sql:ParameterizedQuery filterClause, typedesc<record {}> rowType, string[] include = [])
     returns stream<record {}, sql:Error?>|error {
         sql:ParameterizedQuery query = sql:queryConcat(`SELECT `, self.getSelectColumnNames(include), ` FROM `,
         self.tableName, ` AS `, stringToParameterizedQuery(self.entityName), filterClause);
         return self.dbClient->query(query, rowType);
     }
 
+    # Performs an SQL `UPDATE` operation to update multiple records in the database.
+    #
+    # + object - the key-value pairs to be updated (to be used in the SQL `SET` clauses)
+    # + filter - The key-value pairs to be used as the filter (to be used in the SQL `WHERE` clauses)
+    # + return -  `()` if the operation is performed successfully.
+    #             A `ForeignKeyConstraintViolation` if the operation violates a foreign key constraint.
+    #             An `error` if the operations fails due to an other reason.
```

Comment:
```suggestion
    #             An `error` if the operations fail due to another reason.
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)